src/joint.dia.cell.js

```diff
@@ -261,6 +261,27 @@ joint.dia.Cell = Backbone.Model.extend({
         return this;
     },
 
+    getAncestors: function(){
+
+        var ancestors = [],
+            parentId = this.get('parent');
+
+        if(this.collection === undefined)
+            return ancestors;
+
+        while(parentId !== undefined) {
+            var parent = _.find(this.collection.models, function(item){ return item.id === parentId; });
```

Comment:
`this.collection` is a Backbone collection. So we can call directly `this.collection.get(parentId)` here.